### PR TITLE
Adding known issue to z-stream 41126 TELCODOCS-1185

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1185](https://issues.redhat.com//browse/TELCODOCS-1185): 

Version(s):
4.11

Issue:
https://issues.redhat.com/browse/TELCODOCS-1185
Link to docs preview:
https://56220--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-26

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
@cgoncalves can you PTAL the known issue I added

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -3244,6 +3244,11 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.11.26 --pullspecs
 ----
 
+[id="ocp-4-11-26-known-issues"]
+==== Known issues
+
+* This release regresses functionality where SR-IOV secondary network interfaces configured with static MAC address attached to pods may not provide full network connectivity. This issue only impacts SR-IOV Virtual Functions based on Intel Ethernet Network Adapter X710 products (i40e/iavf Linux kernel drivers). For more information, please see link:https://issues.redhat.com/browse/OCPBUGS-513[*OCPBUGS-513*].
```
Comment (cgoncalves) [inline release_notes/ocp-4-11-release-notes.adoc:3250]: Wrong OCPBUGS

**Comment by cgoncalves** [comment]
LGTMT
The bug has meanwhile been fixed and released in OCP 4.11.28 (https://access.redhat.com/errata/RHBA-2023:0775

**Comment by kquinn1204** [comment]
Great @cgoncalves I guess I should make that clear - WDYT ?

**Comment by cgoncalves** [comment]
I don't know if there's any Docs guidance on this. Perhaps we shouldn't start doing that, otherwise we open a precedence where customers expect us to always go back to past z-stream release notes and amend the notes to state that issues have been addressed in newer streams. That is why I suggested earlier to link to the OCPBUGS, it contains all that info. IMO it should be fine as is.

**Comment by kquinn1204** [comment]
Good point thanks Yes I made a mistake with the bug number.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -3244,6 +3244,11 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.11.26 --pullspecs
 ----
 
+[id="ocp-4-11-26-known-issues"]
+==== Known issues
+
+* This release regresses functionality where SR-IOV secondary network interfaces configured with static MAC address attached to pods may not provide full network connectivity. This issue only impacts SR-IOV virtual functions based on Intel Ethernet Network Adapter X710 products (i40e/iavf Linux kernel drivers). For more information, see link:https://issues.redhat.com/browse/OCPBUGS-5139[*OCPBUGS-5139*].
```
Comment (eohartman) [inline release_notes/ocp-4-11-release-notes.adoc:3250]: Change "may" to "might," per [IBM standards.](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may)

**Review by eohartman — CHANGES_REQUESTED** [review]

**Pull request merged**